src/interp/binary-reader-interp.cc

```diff
@@ -645,6 +649,17 @@ bool BinaryReaderInterp::OnError(const Error& error) {
   return true;
 }
 
+wabt::Result BinaryReaderInterp::EndModule() {
```

Comment:
could be `EndElemSection`, but this is fine too.

Reply:
Hmm, I tried that but it doesn't for if there are is no elem section at all.   Should we change the semantics so end for sections are called even if there is none?  I guess not.  I'll add a comment at least.

Reply:
Oh, good point. Yeah, it shouldn't call if the section is omitted.